main.cpp

```diff
@@ -703,7 +724,8 @@ int main(int argc, char *argv[])
     // flag and rate required always
     // for simplicity of the code outputs allowed only with output
     // for single run (avgs from runs likely not needed)
-    G_option_requires(flg.mortality, opt.infected_to_dead_rate, NULL);
+    G_option_requires_all(flg.mortality, opt.infected_to_dead_rate,
+                          opt.mortality_frequency, NULL);
```

Comment:
Shouldn't `opt.first_year_to_die` be here as well since below it is assumed to be set when flg.mortality is set?

Maybe `G_option_collective` is the right tool here.